js/actions.js

```diff
@@ -0,0 +1,89 @@
+var startBtn = $('#startBtn');
```

Comment:
Thank you for the feedback. What does it mean to "wrap everything"?

On Friday, July 1, 2016, Aric Johnson <notifications@github.com> wrote:

> In js/actions.js
> <https://github.com/drerichards/Bartender/pull/1#discussion_r69368029>:
>
> > @@ -0,0 +1,89 @@
> > +var startBtn = $('#startBtn');
>
> Consider wrapping everything so you don't create a bunch of global
> variables
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/drerichards/Bartender/pull/1/files/2e7cab9e7d873e37fdc28e553f45111bd81c0981#r69368029>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ANLDERmTlSP8P3aOX842J9D-1QeWTZf6ks5qRarGgaJpZM4JC0g6>
> .
>